scripts/get_weather.py

```diff
@@ -0,0 +1,24 @@
+import requests, json
+
+BASE = "https://api.openweathermap.org/data/2.5/weather?"
+APIKEY = "fc2822e01a0efcce52cb56212fe44fd5"
+CITY = "Stony Brook"
+URL = BASE + "q="+ CITY + "&appid=" + APIKEY
+response = requests.get(URL)
+
+if response.status_code == 200:
+    data = response.json()
+    main = data['main']
+    temperature = main['temp'] #in Kelvin
+    tempF = (temperature - 273.15) * (9/5) + 32
+    humidity = main['humidity']
+    pressure = main['pressure']
+    report = data['weather']
+    print("City: {}".format(CITY))
+    print("Temperature (F): {}".format(tempF))
+    print("Humidity: {}".format(humidity))
+    print("Pressure: {}".format(pressure))
+    print("Weather Report: {}".format(report[0]['description']))
+else:
```

Comment:
Let's make if statements to check for 400 and 500 type of status codes. The website returning a 400 is very different from a 500. If we are able to let the user know that a 400 was returned, then the user must take action to correct their mistakes. If a 500 is returned then it is a problem on the website's end and nothing much we can do. 

Never group 400s and 500s under one general error handler. The user must know who was the one who messed up (the user or the web server?)